src/shared/components/default-progress-indicator/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import propTypes from './prop-types'
+
+const DefaultProgressIndicator = ({ percent, visible }) => (
+  <div
+    style={{
+      backgroundColor: '#00ffcb',
```

Comment:
This will mean the progress bar will be the same colour across both brands right?

Reply:
It will but its the default, so we'd supply a custom progress indicator with our custom colours.

Reply:
Unless you think it's worth leaving in default progress indicator colour configurability?

Reply:
@libuk

Reply:
@thomasdigby Ah right, as long as it's configurable should be fine.